internal/twirptest/client_test.go

```diff
@@ -257,8 +257,94 @@ func TestClientRedirectError(t *testing.T) {
 	})
 }
 
+func TestClientWithHooks(t *testing.T) {
+	requestPreparedCalled := false
+	requestFinishedCalled := false
```

Comment:
`%s/equestFinished/esponseReceived/g` in this file :)

Reply:
Ah, good catch. 🤦‍♂